js/repl/ReplOptions.js

```diff
@@ -666,7 +694,40 @@ class ExpandedContainer extends Component<Props, State> {
                 />
               </label>
             </AccordionTab>
-
+            <AccordionTab
+              className={styles.section}
+              isExpanded={isAssumptionsTabExpanded}
+              label="Assumptions"
+              onToggleExpanded={this.handleToggleTabExpanded}
+              tabKey="assumptions"
+            >
+              {ASSUMPTIONS_OPTIONS.map(option => {
+                const isChecked =
+                  assumptions[option] === "undefined"
+                    ? false
+                    : assumptions[option];
+                return (
+                  <label className={styles.envPresetRow}>
+                    <LinkToDocs
+                      className={`${styles.envPresetLabel} ${
+                        styles.highlightWithoutUppercase
+                      }`}
+                      section="assumptions"
```

Comment:
We don't have `#assumptions` section on `preset-env`, since `LinkToDocs` is just an anchor element, we can come up with our own hyper link to `assumptions.html#option`, i.e.

```
docs/en/assumptions#objectrestnosymbols
```

Reply:
done